apps/web/src/server/applications/case/s51/applications-s51.controller.js

```diff
@@ -0,0 +1,67 @@
+/** @typedef {import('@pins/express').ValidationErrors} ValidationErrors */
+/** @typedef {import('./applications-s51.session.js').S51Advice} S51Advice */
+
+import { getSessionS51, setSessionS51 } from './applications-s51.session.js';
+
+/** @type {Record<any, {nextPage: string}>} */
+const createS51Journey = {
+	title: { nextPage: 'enquirer' },
+	enquirer: { nextPage: 'method' },
+	method: { nextPage: 'enquiry-details' },
+	'enquiry-details': { nextPage: 'person' },
+	person: { nextPage: 'advice-details' },
+	'advice-details': { nextPage: 'check-your-answers' }
+};
+
+/**
+ * Show pages for creating/editing s51 advice
+ *
+ * @type {import('@pins/express').RenderHandler<{values: Partial<S51Advice> | null}, {}, {}, {}, {step: string}>}
+ */
+export async function viewApplicationsCaseS51Folder(request, response) {
+	response.render(`applications/case-documentation/folder/documentation-folder`);
```

Comment:
this page can be problematic, but it's not part of the ticket, so we can worry about it later.